components/post_view/post_info/post_info.jsx

```diff
@@ -345,13 +275,11 @@ export default class PostInfo extends React.PureComponent {
             );
         }
 
+        // timestamp should not be a permalink if the post has been deleted, is ephemeral message, or is pending
+        const isPermalink = !(isEphemeral || Posts.POST_DELETED === post.state || ReduxPostUtils.isPostPendingOrFailed(post));
```

Comment:
Nit: We can have this check inside the if clause as before.

Reply:
Good catch, that's totally valid and I really appreciate it.  Thanks!

Now updated.